es-vue-base/tests/EsFormMsg.spec.js

```diff
@@ -8,7 +8,7 @@ describe('EsFormMsg', () => {
         const wrapper = mount(EsFormMsg, {
             ...jestVue,
             propsData: {
-                message: 'Test form msg',
+                show: true,
```

Comment:
This is fine as a basic test of the EsFormMsg rendering, but it doesn't quite keep the same functionality as before where `'Test form msg'` was rendered as the message inside it, which is a test more in line with how it'll be used. `EsButton.spec.js` has a good example where you can pass in a `slots` object just before `propsData` here to pass in the message.